src/java-runtime/assets/jdk.installation.tsx

```diff
@@ -147,6 +147,7 @@ export class JdkInstallationPanel extends React.Component<JdkInstallationPanelPr
             <li><a href="command:java.helper.openUrl?%22https%3A%2F%2Fwww.azul.com%2Fdownloads%2Fazure-only%2Fzulu%2F%22" title="Recommended for Microsoft Azure Cloud and Azure Stack applications">Azul Zulu Enterprise build of OpenJDK for Azure</a></li>
             <li><a href="command:java.helper.openUrl?%22https%3A%2F%2Fdevelopers.redhat.com%2Fproducts%2Fopenjdk%2Fdownload%22" title="Red Hat build of OpenJDK">Red Hat build of OpenJDK</a></li>
             <li><a href="command:java.helper.openUrl?%22https%3A%2F%2Fsapmachine.io%2F%22" title="SapMachine - An OpenJDK release maintained and supported by SAP">SapMachine - An OpenJDK release maintained and supported by SAP</a></li>
+            <li><a href="command:java.helper.openUrl?%22https%3A%2F%2Faws.amazon.com%2Fcorretto%2F%22" title="Amazon Corretto - No-cost, multiplatform, production-ready distribution of OpenJDK">Amazon Corretto - No-cost, multiplatform, production-ready distribution of OpenJDK</a></li>
```

Comment:
Could you please make the description more neutral? Please reference other entries, which are in the style of <vendor> + "OpenJDK".

Reply:
Thanks for your quick answer!

I'm a little confused. What do you mean by more neutral? I don't see how the new description differs from e.g. *"SapMachine - An OpenJDK release maintained and supported by SAP"*.

I proposed *"Amazon Corretto - No-cost, multiplatform, production-ready distribution of OpenJDK"*. From my point of view it is `vendor` (i.e. `Amazon Corretto`) plus `... OpenJDK`. I can change it to *"Amazon Corretto -  An OpenJDK release maintained and supported by Amazon"* if you like that better. That would it make the same like the SapMachine description, but I think a little differentiation would be fine :) and *"No-cost, multiplatform, production-ready distribution of OpenJDK"* is the slogan we usually use for Corretto.

Do you have a better suggestion or can you tell me what specifically you don't like?

Reply:
"Amazon Corretto - An OpenJDK release maintained and supported by Amazon"
This will work for sure. Please go ahead and we can expect a refresh of the Java Extension Pack within this month.

Reply:
Thanks a lot for your quick help. I've updated it now to "Amazon Corretto - An OpenJDK distribution maintained and supported by Amazon" because I think "distribution" is more accurate.

Hope that's still fine :)